addons/mail/models/mail_channel.py

```diff
@@ -105,6 +105,7 @@ def default_get(self, fields):
     image_128 = fields.Image("Image", max_width=128, max_height=128, default=_get_default_image)
     is_subscribed = fields.Boolean(
         'Is Subscribed', compute='_compute_is_subscribed')
+    anonymous_name = fields.Char('Anonymous Name')
```

Comment:
Maybe a good opportunity to add a help attribute explaining this field usage in a small sentence?